core/src/main/scala/org/bitcoins/core/protocol/ln/node/NodeId.scala

```diff
@@ -9,9 +9,6 @@ import scodec.bits.ByteVector
   * 33 byte compressed secp256k1 public key.
   */
 case class NodeId(pubKey: ECPublicKey) extends NetworkElement {
-  require(
```

Comment:
Same here, i guess i don't see why this is inherently safe now without this invariant

Reply:
Same reason ^